kml/serdes.cpp

```diff
@@ -984,9 +976,13 @@ void KmlParser::Pop(std::string_view tag)
     // loading of KML files which were stored by older versions of OMaps.
     TrackLayer layer;
     layer.m_lineWidth = m_trackWidth;
-    layer.m_color.m_predefinedColor = PredefinedColor::None;
-    layer.m_color.m_rgba = (m_color != 0 ? m_color : kDefaultTrackColor);
-    m_trackLayers.push_back(std::move(layer));
+    // Fix wrongly parsed transparent color, see https://github.com/organicmaps/organicmaps/issues/5800
+    // TODO: Remove this fix in 2024 when all users will have their imported GPX files fixed.
+    if (m_color == 0 || (m_color & 0xFF) < 10)
```

Comment:
Is it possible that unitialized value was random? Do we really need it or users can just re-import their tracks?

Reply:
In theory, everything is possible. A random non-transparent value means the track is visible, so it is OK.
Asking users to re-import is not a good idea. Do you see any risks in this fix? We can remove it after a few releases.